nx/lib/nx/defn.ex

```diff
@@ -331,13 +331,9 @@ defmodule Nx.Defn do
 
             #{inspect(arg_template)}
```

Comment:
Should we still put the "Expected" vs "Argument" part above given the new diff implementation?